### PR TITLE
Fix some translations for 2.0 round3

### PR DESCRIPTION
**Pull request by ger-benjamin**
Ref to camptocamp/c2cgeoportal#1834

### CODE REVIEW

File: contribs/gmf/apps/mobile/index.html [hunk 1]
```diff
@@ -74,7 +74,7 @@
           </gmf-layertree>
         </ul>
       </div>
-      <div id="background" class="slide" data-header-title="Background">
+      <div id="background" class="slide" data-header-title="{{'Background' | translate}}">
```
Comment (ger-benjamin) [inline contribs/gmf/apps/mobile/index.html]: Not sur if that will work...

File: contribs/gmf/apps/mobile/index.html [hunk 2]
```diff
@@ -56,16 +56,16 @@
...
       <header>
-        <a class="go-back" href="#"></a>
+        <a class="go-back" href></a>
       </header>
       <!-- main menu -->
       <div class="active slide">
         <ul>
           <li>
-            <a href data-target="#background" data-toggle="slide-in" translate>Background</a>
+            <a href data-target="#background" data-toggle="slide-in">{{'Background' | translate}}</a>
```
Comment (ger-benjamin) [inline contribs/gmf/apps/mobile/index.html]: This change is necessary to (quick) fix the "invisible" submenu effect. But I'm not sur that we can still collect these strings to translate them...

File: contribs/gmf/apps/mobile/index.html [hunk 3]
```diff
@@ -74,16 +74,16 @@
...
-      <div id="background" class="slide" data-header-title="Background">
+      <div id="background" class="slide" data-header-title="{{'Background' | translate}}">
         <gmf-mobile-backgroundlayerselector
           gmf-mobile-backgroundlayerselector-map="::mainCtrl.map"
           gmf-mobile-backgroundlayerselector-select="mainCtrl.hideNav()">
         </gmf-mobile-backgroundlayerselector>
       </div>
       <gmf-themeselector
-        id="themes"
+        id="themes
```
Comment (sbrunner) [inline contribs/gmf/apps/mobile/index.html]: typo

**Comment by sbrunner** [comment]
Just a typo also that's looks good :-)

**Pull request merged**